server/docs/getting-started.md

```diff
@@ -2,106 +2,251 @@
+# Getting Started
 
 ----------
 
-# HighRoller.js
 
-## Truffle unbox
+## Counterfactual's Truffle box
 
 
-We’ll start our new Counterfactual project with the template in the Counterfactual Truffle box. After installing [Truffle](https://truffleframework.com/tutorials/pet-shop), unbox the Counterfactual Truffle box:
+We’ll start our new Counterfactual project with the template in the Counterfactual Truffle box. After installing [Truffle](https://truffleframework.com/docs/truffle/getting-started/installation), and familiarizing yourself with how to [create a new Truffle project](https://truffleframework.com/docs/truffle/getting-started/creating-a-project), unbox the Counterfactual Truffle box:
 
 ```bash
 mkdir my-project
 cd my-project
 truffle unbox counterfactual/truffle-box
 ```
 
-Looking through the template, you’ll find:
+In the box, you'll find `src/app.js`. In this UI template, you'll find: 
 
 * some initialized variables ( `let web3Provider, nodeProvider;` )
 * the async function `run()` which contains calls to
-    * initWeb3() // initializes web3; already complete and explained inline
-    * initContract() // **this is where we point to the blockchain contract for our state channel** - we’ll explain and fill in the details during the guide
-    * setupCF() // setup for the Counterfactual NodeProvider; already complete
-    * install() // **this is the game** - we’ll explain and fill this in
-        * The install function will call the rest of the functions in the doc
+    * initWeb3() // initializes web3; no need to touch this
+    * initContract() // use Truffle to create a Contract object, which enables dynamic fetching of the contract address and content
+    * setupCF() // setup for the Counterfactual NodeProvider; no need to touch this
+    * install() // **this is where we'll code the application UI**
 * a call to the `run()` function
 
+Throughout this guide, we'll be using this template as a reference for developing `app.js`.
+
 ----------
 
 ## Constants
 
-We’ll import some [ethers.js](https://docs.ethers.io/ethers.js) [constants and utilities](https://docs.ethers.io/ethers.js/html/api-utils.html) we’ll need to write High Roller:
+We’ll import some [ethers.js](https://docs.ethers.io/ethers.js) [constants and utilities](https://docs.ethers.io/ethers.js/html/api-utils.html) we’ll need to write **HighRoller**:
 
 * HashZero // the ethers.js bytes32 representation of zero
 * bigNumberify // returns Big Number types from input; we’ll use these because JavaScript is, by default, not able to handle big number representations accurately
 * parseEther // converts the string representation of Ether into BigNumber instance of the amount of Wei
 * solidityKeccak256 // solidity hash function
 * fromExtendedKey // creates an ethereum wallet-like object, [HDNode](https://docs.ethers.io/ethers.js/html/api-advanced.html) from an extended private or public key
```

Comment:
Many of these methods immediately add complexity and I understand why it is at the top of the guide (because these are the import statements) but it does immediately introduce the reader to many concepts that are not necessary for the core CF app experience.